content/riak/kv/2.0.0/developing/usage/conflict-resolution.md

```diff
@@ -541,11 +541,13 @@ obj.store
 bucket = client.bucket_type('siblings_allowed').bucket('nickolodeon')
 obj = bucket.get('best_character')
 
-# Then we modify the object's value
-new_obj.data = 'Stimpy'
+# Then pick one of the siblings
+resolved_sibling = obj.siblings[3]
 
-# Then we store the object, which has the vector clock already attached
-new_obj.store(vclock=vclock)
+# Then replace the siblings data structure with the single sibling we want to keep
+obj.siblings = [resolved_sibling]
+obj.store()
+obj.reload()
```

Comment:
So, I assumed reload would rebuild the obj as if it didn't have siblings. Would this not be something we would want our users to do should they continue to use the object on this fetch?